tchannel/zipkin/tracers.py

```diff
@@ -40,13 +40,17 @@
 
 from .formatters import json_formatter
 from .formatters import thrift_formatter
+from .formatters import i64_to_base64
 from .thrift import TCollector
 from .thrift import constants
+from ..thrift import client_for
 
 log = logging.getLogger('zipkin_tracing')
 
 zipkin_log = logging.getLogger('zipkin')
 
+TCollectorClient = client_for('tcollector', TCollector)
```

Comment:
TCollector and Thrift are already tightly coupled right now.

In fact, we currently have a hard-dependency on the thrift library regardless of whether you use Thrift or not. We need to figure that situation out at some point.